front/lib/api/assistant/generation.ts

```diff
@@ -468,6 +468,9 @@ export async function constructPromptMultiActions(
     }
   }
 
+  additionalInstructions +=
+    "If asked to generate a formula in latex, put the formula between $$.";
```

Comment:
You're missing some `\n` I think? 

"If asked to generate a formula" > "When generating latex math formulas, rely on the $$ escape sequence, single $ latex sequence are not supported" ?

Reply:
Much better, applied 👍🏻